content/applications/general.rst

```diff
@@ -16,130 +16,70 @@ General
-in use.
-
-Odoo is many things (ERP, CMS, CRM application, e-Commerce backend, etc.)
-but it is *not* a smartphone. You should apply caution when adding/removing
-features (especially Apps) on your database since this may impact your
-subscription amount significantly (or switch you from a free account
-to a paying one on our online platform).
-
-This section contains some information as to how you can manage your Odoo
-instances. Before carrying any of these procedures, we **strongly** advise
-to test them on a duplicate of your database first. That way, if something goes
-wrong, your day-to-day business is not impacted.
-
-.. note:: You can find guides on how to duplicate your databases both for
-    :ref:`online <duplicate_online>` and :ref:`on premise <duplicate_premise>`
-    installations.
-
-.. note:: If you have questions about the content of this section or if you
-    encounter an issue while carrying out these procedures, please contact
-    us through our `support form <https://www.odoo.com/help>`__.
-
-
-Deactivating Users
-------------------
-
-.. note:: Make sure you have sufficient **administrative rights** if
-    you want to change the status of any of your users.
-
-In your Odoo instance, click on **Settings**. You will have a section
-showing you the active users on your database. Click on **Manage Users.**
-
-+----------------+----------------+
-|   |settings|   | |browse_users| |
-+----------------+----------------+
-
-You'll then see the list of your users.
-
-.. image:: general/list_users.png
-    :align: center
-
-.. note:: The pre-selected filter *Internal Users* shows your paying
-    users (different from the *Portal Users* which are free). If you
-    remove this filter, you'll get all your users (the ones you pay for
-    and the portal ones)
-
-In your list of users, click on the user you want to deactivate. As soon
-as you are on the userform, click on the Action drop down menu, and
-then click on Archive.
-
-.. image:: general/deactivate_user.png
-    :align: center
-
-
-The user is now deactivated.
-
-.. danger:: **Never** deactivate the main user (*admin*)
-
-Uninstalling Apps
------------------
-
-.. warning:: Make sure you first test what you are about to do on a
-    :ref:`duplicate <duplicate_online>` of your database before making any
-    changes (*especially* installing/uninstalling apps).
-
-In your Odoo instance click on **Settings**; in this app, you will be
-able to see how many applications you have installed. Click on **Browse
-Apps** to access the list of your installed applications.
-
-+----------------+----------------+
-|   |settings|   | |browse_apps|  |
-+----------------+----------------+
-
-In your applications' dashboard, you will see all the icons of your
-applications. Click on the application you want to uninstall. Then, on
-the form of the application, click on **Uninstall**.
+**Odoo** is a suite of open source business apps that cover all your company needs: :doc:`CRM
```

Comment:
```suggestion
**Odoo** is a suite of open source business apps covering all your company needs: :doc:`CRM
```

Reply:
I took the text from the footer on odoo.com .
I'll leave it like that if it's not incorrect.